docker/php/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
```

Comment:
Not sure really; I can't really merge this with the `build.sh` in the root dir, as it needed to be split between the composer and node containers.